packages/ui/src/components/Main/Downloaded.tsx

```
@@ -0,0 +1,134 @@
+
+const List = () => {
+  const { data = [] } = useSuspenseQuery({
+    queryKey: ['downloaded'],
+    queryFn: getDownloaded,
+  });
+
+  return (
+    <Table.Root>
+      <Table.Body>
+        {data.map(({ info_dict: { id, title }, total_bytes }) => (
+          <ListItem id={id} title={title} total_bytes={total_bytes} />
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)